code/modules/mob/living/carbon/slime/life.dm

```diff
@@ -228,7 +228,7 @@
 
 				if(ishuman(L)) //Ignore slime(wo)men
 					var/mob/living/carbon/human/H = L
-					if(H.species.name == "Slime")
+					if(H.species.name == SPECIES_SLIME || H.species.name == SPECIES_TAINTED)
```

Comment:
So everyone with a cruciform will be untargeted by slimes?

Reply:
Depends on taint level